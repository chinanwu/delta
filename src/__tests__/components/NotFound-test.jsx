import { shallow } from 'enzyme';
import React from 'react';

import { NotFound } from '../../components/NotFound.jsx';

jest.unmock('../../components/NotFound.jsx');

describe('NotFound component', () => {
	describe('rendering', () => {
		it('renders NavBar', () => {
			const wrapper = shallow(<NotFound />);
			expect(wrapper.getElement()).toMatchSnapshot();
		});
	});
});
