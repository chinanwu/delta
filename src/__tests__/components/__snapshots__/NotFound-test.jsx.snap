// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component rendering renders NavBar 1`] = `
<div>
  404!
</div>
`;
